index.bs

```diff
@@ -945,6 +945,24 @@ partial interface HTMLIFrameElement {
     <ol>
       <li>Let <var>context</var> be <var>document</var>'s <a>browsing
       context</a>.</li>
+      <li>If <var>context</var> is an [=auxiliary browsing context=]:
+        <ol>
+          <li>Let <var>opener</var> be <var>context</var>'s [=opener browsing
+          context=]'s [=active document=].</li>
+          <li>If either <var>context</var> is not <a
```

Comment:
IIUC, this suggestions means that a *disowned* auxiliary browsing context with a sandbox-ed opener (that has the sandbox flag to propagate sandbox to popups set) will *not* inherit feature policy state. This sounds good but is a different behavior from what sandbox currently does and hence I wonder if it will block (https://github.com/WICG/feature-policy/blob/master/sandbox.md) in its current form.

Should we instead special case sandbox policies here, or for now make feature policy follow sandbox guidelines when sandbox is present.

Reply:
Taking another look I realized this instruction is suggesting what I mentioned above; to give an example,
Suppose feature `sync-xhr` is disabled inside a browsing context. Then the following table maps the combination of `sandbox` and `rel` to the feature state in the auxiliary context (i.e., results of `document.policy.allowsFeature('sync-xhr')`):

| `<iframe>` has:  \\ `window` is:  	|   `rel='noopener'`	|  `rel=''` 	|
|---	|---	|---	|
|  `sandbox` present 	|  `false` 	|   `false`	|
|  `sandbox` present, `allow-popups-to-escape-sandbox` present	|  `true` 	|  `true` 	|
|  `sandbox` not present 	|  `true` 	|  `false` 	|